source/bringauto/modules/mission_module/devices/AutonomyDevice.cpp

```diff
@@ -2,25 +2,36 @@
 #include <fleet_protocol/module_maintainer/module_gateway/module_manager.h>
 
 #include <bringauto/modules/mission_module/devices/AutonomyDevice.hpp>
+#include <bringauto/modules/mission_module/Constants.hpp>
 #include <bringauto/protobuf/ProtobufHelper.hpp>
 #include <google/protobuf/util/message_differencer.h>
 
 
 
 namespace bringauto::modules::mission_module::devices {
 
-int AutonomyDevice::send_status_condition(const struct buffer current_status, const struct buffer new_status) {
+/**
+ * @brief Map of last sent status timestamps for each device type
+ */
+static std::map<unsigned int, std::chrono::milliseconds> last_sent_status_timestamps_ {};
```

Comment:
PLease let the symbol as part of the class.

Reply:
declaration moved to header